src/util/fields.py

```diff
@@ -1191,20 +1191,43 @@ def _update_field_usage_multi(cr, models, old, new, domain_adapter=None, skip_in
                )
     """.format(col_prefix=col_prefix, name=get_value_or_en_translation(cr, "ir_act_server", "name"))
     cr.execute(q, {"old_pattern": p["old_pattern"], "old": p["old"], "standard_modules": tuple(standard_modules)})
+    li = ""
     if cr.rowcount:
-        li = "".join(
+        li = li.join(
             "<li>{}</li>".format(get_anchor_link_to_record("ir.actions.server", aid, aname))
             for aid, aname in cr.fetchall()
         )
+
+    # Modifying compute methods is similarly as dangerous.
+    # Only report potential compute methods that may need an update.
+    q = """
```

Comment:
The `compute` fields has been added in 9.0
Be compatible with upgrade of older databases.
```suggestion
    if column_exists(cr, "ir_model_fields", "compute"):  # for db < 9.0
        # Modifying compute methods is similarly as dangerous.
        # Only report potential compute methods that may need an update.
        q = """
```